src/include/duckdb/execution/operator/persistent/buffered_csv_reader.hpp

```diff
@@ -36,37 +43,81 @@ struct TextSearchShiftArray {
 	unique_ptr<uint8_t[]> shifts;
 };
 
+enum QuoteRule { QUOTES_RFC = 0, QUOTES_OTHER, NO_QUOTES };
+
+enum ParserMode { PARSING = 0, SNIFFING_DIALECT, SNIFFING_DATATYPES };
+
+enum JumpType { START = 0, NEXT };
```

Comment:
Please use an enum class for all of these to avoid polluting the global namespace, i.e.

```cpp
enum class QuoteRule : uint8_t {
...
};
```

The variables must then be accessed through e.g. `QuoteRule::QUOTES_RFC`, rather than only `QUOTES_RFC`.

Also please rename START and NEXT to `JUMP_START` and `JUMP_NEXT`, too generic names can lead to future problems because C developers whose libraries we want to use might include beautiful snippets like `#define START` in their code :)

Reply:
Changed! Btw. JumpType could be dropped, it was not in use anymore.